mesonbuild/compilers/mixins/visualstudio.py

```diff
@@ -458,6 +458,11 @@ def get_instruction_set_args(self, instruction_set: str) -> T.Optional[T.List[st
     def get_pch_base_name(self, header: str) -> str:
         return os.path.basename(header)
 
+clangcl_color_args: T.Dict[str, T.List[str]] = {
+    'auto': ['-fcolor-diagnostics'],
+    'always': ['-fcolor-diagnostics'],
+    'never': ['-fno-color-diagnostics'],
+}
```

Comment:
I would probably just move this into the function since it is only used once

Reply:
I might also add colorout support separately since it is again a commit that could just be pulled in

Reply:
NACK to moving it into a function, the current approach matches the style of other mixins -- to keep that table as a module-global dictionary.

But it could be `from .clang import clang_color_args` instead.